umm-spec-lib/resources/json-schemas/variable/umm/v1.9/umm-var-search-results-json-schema.json

```diff
@@ -0,0 +1,286 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "definitions": {
+    "ItemType": {
+      "type": "object",
+      "additionalProperties": false,
+      "description": "Represents a single item found in search results. It contains some metadata about the item found along with the UMM representing the item. umm won't be present if the item represents a tombstone.",
+      "properties": {
+        "meta": {
+          "$ref": "#/definitions/MetaType"
+        },
+        "umm": {
+          "$ref": "umm-var-json-schema.json"
+        },
+        "associations": {
+          "$ref": "#/definitions/AssociationsInItemType"
+        }
+      },
+      "required": ["meta"]
+    },
+    "MetaType": {
+      "type": "object",
+      "additionalProperties": false,
+      "description": "CMR level metadata about the item found. This represents data not found in the actual metadata.",
+      "properties": {
+        "provider-id": {
+          "description": "The identity of the provider in the CMR",
+          "type": "string",
+          "minLength": 1,
+          "pattern": "[A-Z0-9_]+"
+        },
+        "concept-type": {
+          "description": "The type of item found.",
+          "type": "string",
+          "enum": ["variable"]
+        },
+        "native-id": {
```

Comment:
is this the first time we have included native-id in a metadata document? is this really what we want to do? is it needed? I thought native-ids were suppose to only be used by the provider and even aided in preventing change

Reply:
This file isn't a metadata document, it is for comparing test results when our tests are testing variable search results.
in the UMM_JSON search results format we return the native-id to the caller, this file just tests that it is being returned in the search result format.